lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -174,6 +178,14 @@ def store(self, key, value):
             value
         )
 
+    def store_in_file(self):
```

Comment:
Do not repeat. This is being handled in https://github.com/Seagate/cortx-prvsnr/blob/151c7d164ea4aae589a4a43fdc533da791883e0c/srv/components/system/storage/enclosure_id/config/set.sls#L22
If it is not setting the values in file, update the module code instead.

@tanujashinde0405 you have a better understanding of this part, need your help here.

Reply:
@ypise we can't use set.sls bcoz the fetch_enclosure_id method requires the ip, user, secret and port to be available in the pillar which is not the case during factory storage config.